test/account_api_test.rb

```diff
@@ -0,0 +1,407 @@
+require "test_helper"
+require "gds_api/account_api"
+
+describe GdsApi::AccountApi do
+  include PactTest
+
+  let(:api_client) { GdsApi::AccountApi.new(account_api_host) }
+
+  let(:authenticated_headers) { { "GOVUK-Account-Session" => govuk_account_session } }
+  let(:govuk_account_session) { "logged-in-user-session" }
+
+  describe "getting a sign-in URL" do
+    let(:redirect_path) { nil }
+
+    before do
+      account_api
+        .given("no particular state")
+        .upon_receiving("a sign-in request")
+        .with(
+          method: :get,
+          path: "/api/oauth2/sign-in",
+          headers: GdsApi::JsonClient.default_request_headers,
+        )
+        .will_respond_with(
+          status: 200,
+          headers: { "Content-Type" => "application/json; charset=utf-8" },
+          body: {
+            auth_uri: Pact.like("string"),
+            state: Pact.like("string"),
```

Comment:
I think [Pact matching convention](https://docs.pact.io/getting_started/matching/) is to put an example value here, rather than describing the type. Pact's own matching will infer that the example value is a string.